sqlglot/errors.py

```diff
@@ -30,7 +31,7 @@ class OptimizeError(SqlglotError):
     pass
 
 
-def concat_errors(errors, maximum):
+def concat_errors(errors: t.Union[ErrorLevel, str], maximum: int) -> str:
```

Comment:
I don't think `Union[ErrorLevel, str]` is right. I think it should be `Sequence[Any]`

Reply:
Oops, ErrorLevel is definitely wrong there. Although does `Sequence` capture exceptions? For example, in the parser we call `concat_errors` with `self.errors`, which can have a type of `List[ParseError]`.

Reply:
`Sequence` is an abstract type. I think it works for both List and Tuple? I might have that wrong...

But in either case, the element type is Any. We simply call `str` on the elements, which I think works for all objects.

Reply:
Yeah you're right, I got confused. Will fix this too.